pkgs/misc/cbeams/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, blessings, docopt }:
+
+buildPythonPackage rec {
+  pname = "cbeams";
+  version = "1.0.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0y5q2d4kfrlihj00a4sfwjn9g6a0bwlmnv22xssyldyhlkj1p0aj";
+  };
+
+  propagatedBuildInputs = [ blessings docopt ];
+
+  postPatch = ''
+    sed -i "s/docopt==0.6.1/docopt/; s/blessings==1.6/blessings/" setup.py
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/tartley/cbeams";
+    description = "I've seen things you people wouldn't believe.";
```

Comment:
Descriptions should be descriptive. Even people familiar with the quote won't be able to tell what this is about. Check [Repology](https://repology.org/project/python:cbeams/information) for example.

Reply:
@jtojnar: Thanks for your feedback. I have changed the description from the repository's one to an adaption of the AUR's description.